mergify_engine/gh_pr_fullifier.py

```diff
@@ -172,8 +172,15 @@ def compute_status_desc(pull, **extra):
     return pull.mergify_engine["weight_and_status"][1]
 
 
+def compute_base_is_modifiable(pull, **extra):
+    return (pull.raw_data["maintainer_can_modify"] or
+            (pull.raw_data["head"]["repo"]["id"]
+             == pull.raw_data["base"]["repo"]["id"]))
```

Comment:
for all of this, you can use `pull.head.repo.id` or `pull.maintainer_can_modify`

Reply:
Not sure this should be in the cache, that can be computed, what ever the event and without quering Github.

Reply:
You mean those infos are in the event? Ok, I'll check that out.

Reply:
So GitHub does not send the `maintainer_can_modify` attribute in pull requests event. That means caching this is still needed AFAIU.

Reply:
GitHub does send the maintainer_can_modify attribute in all events: https://developer.github.com/v3/activity/events/types/#pullrequestevent

Reply:
Obviously, the only events I looked into where pull request review which does not have those. Sigh. Thanks @sileht.